tests/unit/test_kinesis.py

```diff
@@ -44,15 +48,19 @@ def test_random_error_on_put_records(self):
         self.assertEqual(failed_record['ErrorMessage'], 'Rate exceeded for shard X in stream Y under account Z.')
 
     def test_overwrite_update_shard_count_on_error(self):
-        update_shard_count_header = {'X-Amz-Target': 'Kinesis_20131202.UpdateShardCount'}
-        request_data = '{"StreamName": "TestStream", "TargetShardCount": 2, "ScalingType": "UNIFORM_SCALING"}'
-        error_response = Response()
-        error_response.status_code = 400
+        if config.KINESIS_PROVIDER == 'kinesalite':
```

Comment:
I only want the proxy request for this to run for kinesalite. So this just returns true if the KINESIS_PROVIDER is kinesis-mock